README.md

```diff
@@ -167,6 +167,7 @@ The runtime environment sets two variables by default
 
 ### Other notes
 
+* Around 2022.10.14, the cuda-nvcc was pinned to 10.6.124, to match the latest cuda version (11.6) that could be installed on the m2lines hubs. This was done to make sure that JAX would work without throwing an error (https://github.com/pangeo-data/pangeo-docker-images/issues/387). 
```

Comment:
10.6.124 -> 11.6.124 ?

Reply:
```suggestion
* Around 2022.10.14, the cuda-nvcc was pinned to 10.6.124, to match cudatoolkit version 11.6 that could be installed on the m2lines hubs. This was done to make sure that JAX would work without throwing an error (https://github.com/pangeo-data/pangeo-docker-images/issues/387). 
```

Probably be specific that 11.6 refers to the cudatoolkit version (yes, the terminology around cuda drivers, cudatoolkit and cuda-nvcc is confusing...).

Do we want to mention something about the minimum CUDA driver version required (e.g. `450.36.06`) even though it's a bit irrelevant for these docker images?

Reply:
I don't think we should make a specific reference to the m2lines hub. Instead, may I suggest something about *avoiding* K80s, as they are a bit old and need version matching per https://github.com/pangeo-data/pangeo-docker-images/issues/390#issuecomment-1278395143? I also am not sure if putting cudatoolkit (or cuda-nvcc?) to latest and testing on T4 would help. Maybe the forward compatibilty measured in https://github.com/pangeo-data/pangeo-docker-images/issues/390#issuecomment-1278395143 means it would be fine? At which point we can just say 'K80 not supported' and be done.